plenum/server/view_change/node_view_changer.py

```diff
@@ -1,11 +1,8 @@
-import logging
-from typing import Set
-
-from plenum.common.messages.internal_messages import NeedViewChange
+from plenum.common.messages.internal_messages import VoteForViewChange
 from plenum.common.startable import Mode
 from plenum.server.quorums import Quorums
+from plenum.server.suspicion_codes import Suspicion
 from plenum.server.view_change.view_changer import ViewChanger, ViewChangerDataProvider
-from storage.kv_store import KeyValueStorage
 
 
```

Comment:
Why can't we just fully remove the class and move all the logic to ViewChangeTriggerService?
It looks like ViewChangeTriggerService needs to be responsible for
- View change by freshness
- Forced view changes
- View Changes on primary disconnection
- View Changes on master degradation

Reply:
I agree that it is much better be handled just by one service. Problem is that this logic is much more tangled with node code, so I decided to postpone it a bit until I resolve problems with main part - sending and handling InstanceChange messages themselves.